pkg/controller/migplan/storage.go

```diff
@@ -13,6 +13,10 @@ func (r ReconcileMigPlan) ensureStorage(plan *migapi.MigPlan) error {
 	var client k8sclient.Client
 	nEnsured := 0
 
+	if plan.Status.HasCriticalCondition() || plan.Status.HasAnyCondition(Suspended) {
+		plan.Status.StageCondition(StorageEnsured)
```

Comment:
Does setting the storageEnsured condition here prevent 'storageEnsure' from running?

Reply:
No, it just preserves the `StageEnsured` condition if it already exists.  The `Suspended` condition prevents the _insure storage logic_ from running.